dashboard/src/containers/Routes.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { mount } from "enzyme";
+import * as React from "react";
+import { MemoryRouter } from "react-router";
+import NotFound from "../components/NotFound";
+import Routes from "./Routes";
+
+it("invalid path should redirect to 404", () => {
+  const wrapper = mount(
+    <MemoryRouter initialEntries={["/random"]}>
```

Comment:
In other projects, for testing, I've been using staticRouter instead of MemoryRouter, I am curious what's your take on this and why you decided to choose this one?

Reply:
it was the one I found googling about the issue. It's true that we are not changing the URL in the test so staticRouter should be enough (MemoryRouter keeps track of the URLs change)

Reply:
Thanks for the explanation.